src/web/screens/apiManagement.tsx

```diff
@@ -0,0 +1,40 @@
+import { Suspense } from 'react';
+import { Await, defer, useLoaderData } from 'react-router-dom';
+
+import { ApiKeyDTO } from '../../api/services/adminServiceHelpers';
+import KeyTable from '../components/ApiKeyManagement/KeyTable';
+import { Loading } from '../components/Core/Loading';
+import { GetParticipantApiKeys } from '../services/participant';
+import { RouteErrorBoundary } from '../utils/RouteErrorBoundary';
+import { PortalRoute } from './routeUtils';
+
+function ManageParticipants() {
+  const data = useLoaderData() as {
+    result: ApiKeyDTO[];
+  };
+
+  return (
+    <div>
+      <h1>Manage API Keys</h1>
+      <p className='heading-details'>View and manage you API keys.</p>
+      <Suspense fallback={<Loading />}>
+        <Await resolve={data.result}>
+          {(apiKeys: ApiKeyDTO[]) => <KeyTable apiKeys={apiKeys.filter((key) => !key.disabled)} />}
+        </Await>
+      </Suspense>
+    </div>
+  );
+}
+
+export const ApiManagementRoute: PortalRoute = {
+  path: '/dashboard/api',
```

Comment:
I think this should probably match the file name too. `api` is a bit confusing because it's not an API per -e.